2021q3/freebsd-foundation.adoc

```diff
@@ -0,0 +1,169 @@
+
+We also have the Partnership Program, to provide more benefits for our larger
+commercial donors. Find out more information at
+link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/[https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/]
+and share with your companies!
+
+==== OS Improvements
+
+During the third quarter, Foundation staff and grant recipients committed 420
+src tree changes, 24 ports tree changes, and 11 doc tree changes. This
+represents 38% of src commits which identify a sponsor. For doc and ports
+commits the values are 48% and 16%.
+
+You can read about Foundation-sponsored projects in individual quarterly report
+entries:
+
+- Base System OpenSSH Update
+- Fixes for msdosfs_rename VOP
+- Hole Punching
+- Improved amd64 UEFI boot
+- Intel Wireless driver support
+- LLDB Debugger Improvements
+- Microchip LAN743x mgb(4) Device Driver
+- OpenZFS RAIDZ Expansion update
+- Using OCF in WireGuard
+- syzkaller on FreeBSD
+
+==== Supporting FreeBSD Infrastructure
+
+The Foundation provides hardware and support for the Project.  Last quarter, we
+continued supporting the test cluster at Sentex, purchased a few needed drives,
+and spam filtering software for project email. We also set up a new and more
+efficient hardware request/purchase process for clusteradm to use.
+
+==== Partnerships and Commercial User Support
+
+We met virtually with corporate users and started travelling again in late Q3
+for some in-person meetings. The goals of the meetings are to facilitate
+collaboration between commercial users and FreeBSD developers. We also met with
+companies to discuss their needs and share that information with the Project.
+
+==== FreeBSD Advocacy and Education
+
+Much of our effort is dedicated to Project advocacy.  This may involve
+highlighting interesting FreeBSD work, producing literature, attending events,
+or giving presentations.  The goal of the literature we produce is to teach
+people FreeBSD basics and help make their path to adoption or contribution
+easier.  Other than attending and presenting at events, we encourage and help
+community members run their own FreeBSD events, give presentations, or staff
+FreeBSD tables.
+
+The FreeBSD Foundation sponsors many conferences, events, and summits around the
+globe. These events can be BSD-related, open source, or technology events geared
+towards underrepresented groups. We support the FreeBSD-focused events to help
+provide a venue for sharing knowledge, working together on projects, and
+facilitating collaboration between developers and commercial users. This all
+helps provide a healthy ecosystem. We support the non-FreeBSD events to promote
+and raise awareness of FreeBSD, to increase the use of FreeBSD in different
+applications, and to recruit more contributors to the Project. We finally made
+it back to our first in-person meeting with the *Open Source Summit* in late
```

Comment:
"We finally started meeting in person again..." maybe?

Reply:
I feel the same about changing Deb's words on this one.  I don't see a problem with the original.